GSA/made-in-america
# MIA-148 Implements form scroll to top on next and previous navigation

## Description
closes https://cm-jira.usa.gov/browse/MIA-148

implementing the `@4.14.0` [min js from formio](https://github.com/formio/enterprise-release/blob/master/API-Server-Change-Log.md#december-1st-2021-release) did not help resolve this issue as we had hoped.

however, we were able to hook into some of the events and trigger a scroll to the top on `nextPage` and `prevPage` events

## How to test

- pull changes locally
- npm install
- npm run start
- navigate to `/waiver-form`
- click "next" button
- fill out first page of wizard
-   - use code `SP8000`
- click "next" button
- click "previous" button
- in all cases window should scroll to top of window

To prepare for code review, please follow the following checks.
